convert2rhel/unit_tests/actions/system_checks/rhel_compatible_kernel_test.py

```diff
@@ -146,6 +239,48 @@ def test_bad_kernel_substring(kernel_release, exp_return):
             "yajl.x86_64",
             False,
         ),
+    ),
+)
+@centos8
+def test_bad_kernel_package_signature_success(
```

Comment:
Why this was renamed?

Reply:
Also, it seems that `get_installed_pkg_objects` is not used in the test, maybe it is worth it to remove?

Reply:
get_installed_pkg_objects is used to mock information used in the check, without it the test fails